### ISSUE TITLE
Unable to add UWP Test project template in VS 2017

### ISSUE THREAD

**Issue by HappyTreesDev** [nunit/nunit-vs-templates#40]
I am attempting to add NUnit 3 Test Project (Universal Windows) to my solution and I am getting an error every time I try.

I get the error "Unable to copy the file 'project.json' from the project template to the project. Cannot find file "C:\Users\XXXX\AppData\Local\Temp\XXXX\Temp\project.json"

**Comment by rprouse** [comment]
The UWP test project hasn't been updated in awhile. Are you using VS2017? I am not sure it still works there.

**Comment by HappyTreesDev** [comment]
Yes I am using VS 2017. I decided to use a UWP test project built upon MSTest for now.

**Comment by ChrisMaddock** [comment]
I'm fairly sure this worked in the original version of VS 2017 - I wonder if something's changed with the move to `dotnet new`.

This project's becoming hard to keep up with. 😞 I'm starting to wonder if we should revert the current project to max VS 2015, and do something new with VS 2017. My only reason for not doing that - is that I use this in VS 2017 every day!

**Comment by rprouse** [comment]
I've actually stopped using this in 2017. Whenever I use the snippet in .NET Core projects, it gives me errors about the using statements :(

I really want to see NUnit project templates in VS to compete with the MSTest and xUnit templates that are shipped with VS, but it would be nice if we knew what format we should be using to maintain compatibility going forward.

**Comment by sunchunqiang** [comment]
is the issue fixed?

**Comment by rprouse** [comment]
@sunchunqiang no, if the issue was fixed, it would be closed.

**Comment by michaelkollmann** [comment]
Are you still investigating this issue or will this just not be fixed?
Is there a workaround?

**Comment by ChrisMaddock** [comment]
@michaelkollmann - this project is looked after by a small team of volunteers, who are stretched thinly. We’d welcome a pull request to fix this issue.

**Comment by bee-s** [comment]
Any update on this ? I can still see that the issue exists. 
2 years and still stretched out ? I wish i had all the skills to do this myself.
